pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -495,6 +498,7 @@ func (c *Reconciler) updateTaskRunStatusForTimeout(tr *v1alpha1.TaskRun, dp Dele
 		}
 	}
 
+	timeout := tr.Spec.Timeout.Duration
```

Comment:
hm this is interesting cuz it means if the order of the call to this function were to change, this could still be nil 🤔